java/core/src/main/java/co/worklytics/psoxy/impl/RESTApiSanitizerImpl.java

```diff
@@ -116,12 +136,66 @@ public boolean isAllowed(@NonNull String httpMethod, @NonNull URL url) {
         return rules.getAllowAllEndpoints() || getEndpoint(httpMethod, url).isPresent();
     }
 
+    @Override
+    public boolean isAllowed(String httpMethod, URL url, @Nullable String contentType,
+            @Nullable String requestBody) {
```

Comment:
to generalize to support PUT/POST, had to add this.

left the `isAllowed(String, URL)` bc used throughout LOTS of tests; that is implicitly a `GET`/`HEAD`. could/should probably drop it, bc otherwise user of class needs to parse http request method, and pick form of `isAllowed()` based on whether they expect body with content-type for the given http request type or not.